applications/StructuralMechanicsApplication/tests/test_multipoint_contstraints.py

```diff
@@ -108,230 +123,171 @@ def _solve(self):
+        cm.AddMasterSlaveRelation(
+            mp.Nodes[18], KratosMultiphysics.DISPLACEMENT_Y, mp.Nodes[7],
+            KratosMultiphysics.DISPLACEMENT_Y, 0.5, 0)
+
+        cm.AddMasterSlaveRelation(
+            mp.Nodes[18], KratosMultiphysics.DISPLACEMENT_X, mp.Nodes[9],
+            KratosMultiphysics.DISPLACEMENT_X, 1.0, 0)
+        cm.AddMasterSlaveRelation(
+            mp.Nodes[18], KratosMultiphysics.DISPLACEMENT_Y, mp.Nodes[9],
+            KratosMultiphysics.DISPLACEMENT_Y, 1.0, 0)
 
         return mp, cm
 
-    def _set_and_fill_buffer(self,mp,buffer_size,delta_time):
+    def _set_and_fill_buffer(self, mp, buffer_size, delta_time):
         # Set buffer size
         mp.SetBufferSize(buffer_size)
         # Fill buffer
         time = mp.ProcessInfo[KratosMultiphysics.TIME]
         time = time - delta_time * (buffer_size)
         mp.ProcessInfo.SetValue(KratosMultiphysics.TIME, time)
         for size in range(0, buffer_size):
-            step = size - (buffer_size -1)
+            step = size - (buffer_size - 1)
             mp.ProcessInfo.SetValue(KratosMultiphysics.STEP, step)
             time = time + delta_time
             #delta_time is computed from previous time in process_info
             mp.CloneTimeStep(time)
 
-        mp.ProcessInfo[KratosMultiphysics.IS_RESTARTED] = False     
+        mp.ProcessInfo[KratosMultiphysics.IS_RESTARTED] = False
 
         return mp
 
     def test_MPC_Constraints(self):
         dim = 2
-        mp = KratosMultiphysics.ModelPart("solid_part")    
-        mp = self._apply_material_properties(mp,dim)            
+        mp = KratosMultiphysics.ModelPart("solid_part")
+        mp = self._apply_material_properties(mp, dim)
         mp = self._setup_model_part(mp)
-        mp = self._add_variables(mp)        
+        mp = self._add_variables(mp)
 
         #time integration parameters
         dt = 0.005
         time = 0.0
-        end_time = 0.01
+        end_time = 0.025
         step = 0
-        
-        mp = self._set_and_fill_buffer(mp,2,dt)
+
+        mp = self._set_and_fill_buffer(mp, 2, dt)
         # Applying boundary conditions
         mp = self._apply_BCs(mp)
         # Applying constraints
-        cm = KratosMultiphysics.StructuralMechanicsApplication.ApplyMultipointConstraintsProcess(mp)
-        mp, cm = self._apply_mpc_constraints(mp,cm)
+        cm = KratosMultiphysics.StructuralMechanicsApplication.ApplyMultipointConstraintsProcess(
+            mp)
+        mp, cm = self._apply_mpc_constraints(mp, cm)
         # Solving the system of equations        
         self._setup_solver(mp)
 
-        while(time <= end_time):
+        while (time <= end_time):
             time = time + dt
             step = step + 1
-            print('############ Time :: ', time, ' ### step ', step)
             mp.CloneTimeStep(time)
-
-            self._solve()
+            if (step > 3):
+                print('############################### Time :: ', time,
```

Comment:
Protect that by an echo lv?
I think one can set some verbosity in the tests

Reply:
I will remove it I think as even with a echo lvl I dont see it is necessary.!

Reply:
Thanks !